src/dispatch/plugins/dispatch_slack/decorators.py

```diff
@@ -112,17 +168,20 @@ def wrapper(*args, **kwargs):
 
             # we notify the user that the interaction failed
             message = (
-                f"Sorry, we've run into an unexpected error. For help, please reach out to {conversation.incident.commander.individual.name}",
+                "Sorry, we've run into an unexpected error. For help please reach out to your Dispatch admins",
                 f" and provide them with the following token: {slack_interaction_guid}.",
             )
-            if conversation.incident.status != IncidentStatus.closed:
-                dispatch_slack_service.send_ephemeral_message(
-                    kwargs["slack_client"], channel_id, user_id, message
-                )
+
+            if conversation:
+                if conversation.incident.status != IncidentStatus.closed:
+                    dispatch_slack_service.send_ephemeral_message(
+                        kwargs["slack_client"], channel_id, user_id, message
+                    )
             else:
-                dispatch_slack_service.send_message(
+                dispatch_slack_service.send_ephemeral_message(
                     client=kwargs["slack_client"],
-                    conversation_id=user_id,
+                    conversation_id=channel_id,
+                    user_id=user_id,
                     text=message,
                 )
```

Comment:
Looks like we're not messaging the user if we have a conversation and the status of the incident is closed now.

Reply:
afaik this is not new functionality, I simply re-worked the logic to enable error messaging for non-incident commands.

Reply:
Yes, it does change the logic introduced in https://github.com/Netflix/dispatch/pull/1520 to address Slack channel `is_archived` errors.